backports/glog/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Florent Ferreri <florent@seqsense.com>
+pkgname=glog
+pkgver=0.4.0
+pkgrel=0
+pkgdesc="Library and Utilities for working with ifm pmd-based 3D ToF Cameras"
```

Comment:
`pkgdesc` should be a description of this package.

Reply:
Sorry copy paste mistake! :bowing_man: